ios/FluentUI/Navigation/SearchBar.swift

```diff
@@ -419,6 +445,27 @@ open class SearchBar: UIView {
         NSLayoutConstraint.activate(constraints)
     }
 
+    private func setupLayoutForLeadingView() {
+        guard let leadingView = leadingView else {
+            textFieldLeadingConstraint?.constant = Constants.searchTextFieldLeadingInset
```

Comment:
You can just activate/deactivate the constraint to the icon's trailing anchor here.

Reply:
Done